packages/x-components/src/x-modules/tagging/events.types.ts

```diff
@@ -25,4 +28,9 @@ export interface TaggingXEvents {
    * * Payload: The new session duration value.
    */
   SessionDurationProvided: number;
+  /**
+   * Query tagging info has changed.
+   * * Payload: The new query tagging info.
+   */
+  SearchTaggingEmitted: TaggingInfo;
```

Comment:
I'm not sure about the name.

I mean this event will be emitted when the `TaggingInfo` change (at the same time this change is done when the search tagging changes)

![image](https://user-images.githubusercontent.com/3502075/147201267-11397684-f734-4686-927b-a0306e7b3651.png)

I would rather use something like `TaggingInfoChanged`, which for me express better what changes.

What do you think?

Reply:
I was not sure about SearchTaggingEmitted so I just changed it to SearchTaggingReceived which seems to be quite better to me. Anyway, we can have a call tomorrow about it.